README.md

```diff
@@ -207,6 +207,11 @@ Taking a counter as an example:
 ```python
 from prometheus_client import Counter
 c = Counter('my_requests_total', 'HTTP Failures', ['method', 'endpoint'])
+# when using labels, metrics are not initialized at creation
```

Comment:
This doesn't really fit here, as a less common use case is being wedged in front of the simple normal usage.

This would be better as a separate thing down the send of this section.

Reply:
Fair enough, I thought it was a strong enough recommendation to highlight it in the normal use case. Pushed a revision.